src/main/java/com/ncc/aif/ValidateAIFCli.java

```diff
@@ -0,0 +1,597 @@
+package com.ncc.aif;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import com.google.common.base.Charsets;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.io.CharSource;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.riot.RDFDataMgr;
+import org.apache.jena.riot.RDFFormat;
+import org.topbraid.jenax.statistics.ExecStatistics;
+import org.topbraid.jenax.statistics.ExecStatisticsListener;
+import org.topbraid.jenax.statistics.ExecStatisticsManager;
+import org.topbraid.shacl.vocabulary.SH;
+import picocli.CommandLine;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Spec;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.concurrent.Callable;
+
+/**
+ * A command-line AIF validator.  For details, see <a href="https://github.com/NextCenturyCorporation/AIDA-Interchange-Format">the AIF README</a>
+ * section entitled, <i>The AIF Validator</i>.
+ */
+@CommandLine.Command(name = "validateAIF",
+        sortOptions = false,
+        synopsisHeading = "%nUsage: ",
+        descriptionHeading = "%nDescription: ",
+        optionListHeading = "%nOptions:%n",
+        description = "Used to validate Turtle files with extension .ttl",
```

Comment:
Don't we need to mention AIDA/AIF here?  I know it can validate non-AIF, but the fact that it validates against the Interchange ontology is pretty central to its existence.

Reply:
Looks like this is fixed.  Resolving.